lesson9.tex

```diff
@@ -0,0 +1,123 @@
+     Bot Island explaining all the current events that occur on Bot Island. Instructors discuss amongst themselves what
+     strategies they will employ in investing their money. One should invest in all stocks equally (simulating an index fund),
+     another should invest all their money in one stock, and (optionally) another can refuse to put their money into the market.
+    \subsection{Introduction (5 minutes)}
+     Present the first slideshow, which will explain what a stock is, and how a stock exchange consists of all the available stocks
+     in a certain place. Then, explain how we are all new residents of the tropical island called Bot Island, and how there
+     are several companies on the island that each offer stocks. Give each student their set amount of initial BotCash,
+     and ask them if they'd like to participate in the island's stock exchange (appropriately named the B\&B exchange). 
+
+    \subsection{Demonstration}
+     Instructors will also be given their BotCash, and will announce what their strategies are for investing into the stock
+     market (as listed above). They should demonstrate how to submit their form to purchase their stocks.
+     An instructor will then update a pre-made spreadsheet detailing the transactions.
+    \subsection{Guided Practice}
+      Inform students that they are allowed to form groups to make their own "firm", and can discuss amongst themselves
+      which stocks they would like to purchase or sell. Have them submit their forms, and the instructor should update
+      a spreadsheet detailing the transactions that the students made. After all students have made their purchases,
+      the first "round", or the first newspaper, should be shown with the first event. Events should start simply first,
+      perhaps a nice sunny day on the island. These events will be shown as the front pages of newspapers in the
+      slideshow. As students are starting to feel more comfortable making decisions and buying/selling stocks, events 
+     should get more drastic and complicated. Listed are a few examples of events:
+       \begin{itemize} 
+          \item A very sunny day on the island (increases value of stocks in sunscreen stocks, hotel resort stocks, electricity
+                   company, etc. Simultaneously decreases value of umbrella companies, etc.)
+          \item A rare shark is seen off the coast of Bot Island 
+                   (increases value of tourism company stock, decreases the value of the surfing company stock)
+          \item A massive storm hits Bot Island (simulates a crash in the market)
+          \item Bot Island is named the best island of the year (huge jump in all stocks)
+          \item Bot Island gets huge amounts of rainfall (increases value of umbrella company,
+                   decreases value of sunscreen, tourism, hotel resort stocks)
+      \end{itemize}
+      \subsubsection{Sticking points:}
```

Comment:
Would there be a way for us to add a predictive aspect to the lesson? So the kids can hear about possible weather going on, and things like that? Some of the market should be unpredictable, but other parts can be partially predicted, which could be a cool thing to add in.

Reply:
Maybe this could be a bit of media? Like a daily news broadcast with weather report and other random news (some helpful, some not)?

Reply:
I was thinking about making a "daily news" slide for every event listed, with things like the weather being predictive and other things like a shark sighting being unpredictable